### PR TITLE
reaction regression/optimization

### PR DESCRIPTION
**Pull request by bojana-rankovic**
not yet to be merged

### CODE REVIEW

File: gprotorch/data_featuriser/featurisation.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        try:
+            features = [fragments[d](mol) for d in fragments]
+        except:
+            raise Exception("molecule {}".format(i) + " is not canonicalised")
+        frags[i, :] = features
+
+    return frags
+
+
+def chemberta_features(smiles):
```
Comment (Ryan-Rhys) [inline gprotorch/data_featuriser/featurisation.py:63]: Cool! Are these just for reactions?

File: gprotorch/data_featuriser/featurisation.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        for tokenized_smile in tokenized_smiles
+    ]
+
+    embeddings = torch.cat(
+        [output["hidden_states"][0].sum(axis=1) for output in outputs], axis=0
+    )
+    return embeddings.detach().numpy()
+
+
+def graphs(smiles, graphein_config=None):
```
Comment (Ryan-Rhys) [inline gprotorch/data_featuriser/featurisation.py:87]: Ah, do we use Graphein for all our graph featurisation?

File: gprotorch/data_featuriser/featurisation.py (hunk 1 above)
Comment (bojana-rankovic) [reply gprotorch/data_featuriser/featurisation.py:63]: No, this one is actually for molecules

**Review by sangttruong — APPROVED** [review]

File: gprotorch/data_featuriser/featurisation.py (hunk 1 above)
Comment (Ryan-Rhys) [reply gprotorch/data_featuriser/featurisation.py:63]: Ah nice!

**Pull request closed without merging**